### PR TITLE
Feature/sn 454 add properties alphabet ordering

### PR DESCRIPTION
**Pull request by AlexanderMakarov124**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,4 +207,30 @@ services.AddNetForge(optionsBuilder =>
 
 ## Search
 
-You can read about search [here](docs/SEARCH.md)
\ No newline at end of file
+You can read about search [here](docs/SEARCH.md)
+
+## Sort
```
Comment (vasiliy-chefonov) [inline README.md:212]:
```suggestion
## Data Sorting
```

File: demo/Saritasa.NetForge.Demo.Net7/Infrastructure/DependencyInjection/NetForgeModule.cs [hunk 2]
```diff
@@ -29,6 +29,16 @@ public static void Register(IServiceCollection services)
                     {
                         return query.Where(e => e.Name.Contains(searchTerm));
                     });
+
+                entityOptionsBuilder.ConfigureProperty(shop => shop.IsOpen, builder =>
```
Comment (vasiliy-chefonov) [inline demo/Saritasa.NetForge.Demo.Net7/Infrastructure/DependencyInjection/NetForgeModule.cs:33]: Can we move this to separate configuration class?

File: src/Saritasa.NetForge.Mvvm/ViewModels/EntityDetails/EntityDetailsViewModel.cs [hunk 3]
```diff
@@ -56,11 +57,32 @@ public override async Task LoadAsync(CancellationToken cancellationToken)
...
+        var sortDefinitions = gridState.SortDefinitions
+            .Select(sort => new
+            {
+                DataGrid!.RenderedColumns.First(column => column.PropertyName.Equals(sort.SortBy)).Title,
+                sort.Descending,
+                sort.Index
+            })
+            .ToList();
+
+        var orderBy = new StringBuilder();
```
Comment (vasiliy-chefonov) [inline src/Saritasa.NetForge.Mvvm/ViewModels/EntityDetails/EntityDetailsViewModel.cs:69]: Can we just have a separate class / record for storing these sorting options? Then we'll pass the instance to the service. I'm not sure whether the ViewModel should be responsible for such a string transformation.

File: src/Saritasa.NetForge.UseCases/Services/EntityService.cs [hunk 4]
```diff
@@ -112,4 +120,54 @@ public Task<GetEntityByIdDto> GetEntityByIdAsync(string stringId, CancellationTo
...
+    private static IOrderedQueryable<object> Order(IQueryable<object> query, string orderBy, Type entityType)
+    {
+        var separatedOrderBy = OrderParsingDelegates.ParseSeparated(orderBy);
+        var keySelectors = GetKeySelectors(separatedOrderBy, entityType);
+
+        return CollectionUtils.OrderMultiple(query, separatedOrderBy, keySelectors);
+    }
+
+    private static (string FieldName, Expression<Func<object, object>> Selector)[] GetKeySelectors(
+        (string FieldName, ListSortDirection Order)[] orderBy, Type entityType)
```
Comment (vasiliy-chefonov) [inline src/Saritasa.NetForge.UseCases/Services/EntityService.cs:133]: I guess you can only pass the field names to get the selectors

**Review by vasiliy-chefonov — APPROVED** [review]

**Review by toanphamsaritasa — APPROVED** [review]

**Pull request merged**